code/datums/components/machine_booby_trap.dm

```diff
@@ -0,0 +1,102 @@
+/**
+ * Attached to an atom, creates an explosion when it is interacted with
+ */
+/datum/component/
```

Comment:
you haven't given this component a typepath

Reply:
How the hell did I do that